testing/util/blob.go

```diff
@@ -5,6 +5,19 @@ import (
 	ethpb "github.com/prysmaticlabs/prysm/v4/proto/prysm/v1alpha1"
 )
 
+// HydrateSignedBlobSidecar hydrates a signed blob sidecar with correct field length sizes
+// to comply with SSZ marshalling and unmarshalling rules.
+func HydrateSignedBlobSidecar(b *ethpb.SignedBlobSidecar) *ethpb.SignedBlobSidecar {
```

Comment:
None of this blob stuff seems related to your PR

Reply:
I use it in `events_test.go`